### DISCUSSION TITLE
Why overlays don't close on the right click outside?

### DISCUSSION THREAD

**Discussion by vovsemenv** [adobe/react-spectrum#7126] [Q&A]
React-aria popover really great overall but there is no way to close popover when user right click outside of menu. Why we don't close in on left click. for exapmple native select is closing on right click outside https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#try_it

**Comment by reidbarber** [comment]
Discussed this with the team. We like the current behavior since it's 1 action to 1 action. (Right click just opens context menu). If this this is an important feature in your app, you could probably add a custom listener to the underlay to close it on right click.
